src/morphology.cpp

```diff
@@ -137,14 +137,26 @@ const Points& Morphology::points() const
 {
     return get<Property::Point>();
 }
+
+const std::vector<uint32_t> Morphology::sectionOffsets() const
+{
+    const std::vector<std::array<int, 2>>& indices_and_parents = get<Property::Section>();
+    std::vector<uint32_t> indices(indices_and_parents.size());
+    std::transform(indices_and_parents.begin(), indices_and_parents.end(), indices.begin(),
+                   [](const std::array<int, 2>& pair) { return pair[0]; });
```

Comment:
instead of `std::array<int, 2>`, shouldn't it be `Section::Type` - this applies everywhere.